### PR TITLE
Fix poorly done deprecations in image.py.

### PR DESCRIPTION
**Pull request by anntzer**
xref #9332.

## PR Checklist

- [ ] Has Pytest style unit tests
- [ ] Code is PEP 8 compliant 
- [ ] New features are documented, with examples if plot related
- [ ] Documentation is sphinx and numpydoc compliant
- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way

### CODE REVIEW

**Comment by tacaswell** [comment]
took the liberty of adding a test.

**Comment by WeatherGod** [comment]
Is the appveyor failure the same as the one that has been happening?

File: lib/matplotlib/image.py [hunk 1]
```diff
@@ -180,14 +180,17 @@ def _rgb_to_rgba(A):
...
     def _interpd(self):
         return _interpd_
 
+    @property
     @cbook.deprecated("2.1")
     def _interpdr(self):
         return {v: k for k, v in six.iteritems(_interpd_)}
 
+    @property
     @cbook.deprecated("2.1")
```
Comment (WeatherGod) [inline lib/matplotlib/image.py]: Shouldn't this deprecation tell us what to use instead?

Comment (anntzer) [reply lib/matplotlib/image.py]: added

**Pull request merged**